src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -11,6 +11,10 @@
 using System.Reflection;
 using Roslyn.Utilities;
 
+#if NET
+using System.Runtime.Loader;
+#endif
```

Comment:
Don't think this is needed

Reply:
technically needed since i have a `<see cref="AssemblyLoadContext"/>` below in teh comment.  And that type (and namespace) is only available on NET not below.